snippets/word_count.md

```diff
@@ -0,0 +1,29 @@
+---
+title: word_count
+tags: string,beginner
+firstSeen: 2021-07-18T08:14:11+0000
+lastUpdated: 2021-07-18T08:14:11+0000
+---
+
+Calculates the total number of occurence of a word in a given string .
+
+- Uses `.split()` method to give a list of words from the string, which is used to find the total occurence of a word in that list using a counter `count`
+
+```py
+def word_count(query, string):
+  words = string.split()
+  count = 0
+
+  for word in words:
+    if word == query:
+      count += 1
+

```

Comment:
Just a suggestion, I think this is much more simper snippet.

``` py
from collections import Counter

def word_count(query, string):
    words = string.split()
    counter = Counter(words)
    return counter[query]
```

Reply:
Thanks i will update the snippet. It's just that i have not known about this module . Well thanks again .